Lib/glyphsLib/builder.py

```diff
@@ -340,6 +344,10 @@ def set_custom_params(ufo, parsed=None, data=None, misc_keys=(), non_info=()):
         if name.startswith('openTypeOS2Win') and value < 0:
             value = -value
 
+        # Some Glyphs files store weightClass/widthClass as string values.
+        if name in ('openTypeOS2WeightClass', 'openTypeOS2WidthClass'):
+            value = int(value)
```

Comment:
I tried, but it wouldn’t work. The `weightClass` parameter has enum values such as `Bold`, whereas `customParameters.weightClass` has numeric values inside a string. For example, here’s a snippet from the [WeightTest.glyphs](https://github.com/googlei18n/fontmake/files/793138/WeightTest.zip) file posted on the bug:

```
{
  customParameters = ({ name = weightClass; value = "707"; });
  interpolationWeight = 215;
  isBold = 1;
  linkStyle = Regular;
  name = Fat;
  weightClass = Bold;
}
```